### PR TITLE
Added GUI Execution Time

### LINKED ISSUE #19: Display execution time after query in GUI
You cant really tell the difference in execution time due to using threads. We should show how long it took to perform the last query on the GUI. I recommend it go somewhere ontop of the "Best Match" section just as a piece of text

### PR DESCRIPTION
**Pull request by TBean18**
image

### CODE REVIEW

**Review by Armintron — APPROVED** [review]
Nice

**Review by Djhranicky — APPROVED** [review]
Good stuff!

File: src/Main.java [hunk 1]
```diff
@@ -61,6 +69,8 @@ public static void startAndWaitForThreads(Runnable r, int num) {
                 e.printStackTrace();
             }
         }
+        long dur = System.currentTimeMillis() - start;
+        return dur;
```
Comment (conradsmi) [inline src/Main.java:73]: Good stuff - I should probably change runIterations() in Benchmarking so that it makes use of this instead

**Review by conradsmi — APPROVED** [review]
Nice

**Pull request merged**